.github/workflows/.windows.yml

```diff
@@ -361,7 +361,7 @@ jobs:
         run: |
           Write-Host "Creating service"
           If ("${{ matrix.runtime }}" -eq "containerd") {
-            $runtimeArg="--containerd=\\.\pipe\containerd-containerd"
+            $runtimeArg="--default-runtime=io.containerd.runhcs.v1"
```

Comment:
The previous block that starts containerd manually should be removed, since we aren't using that containerd, and it'll cause the newly-added check on line 404-408 to false-pass.

That also applies to the "Stop containerd" step later on (which will fail when "Starting containerd"), and "Containerd logs" needs to either be changed to get the logs from the Docker-managed instance, or removed. (I think the former is better, unless and until we are able to plumb the containerd log data into the docker log data... Not sure if we do that on Linux though, so that might be a new feature.)

Reply:
@TBBle I have removed the manual steps for starting and stopping containerd.

At present, containerd logs are being redirected to Docker logs **_(libcontainerd/supervisor/remote_daemon.go)_**

![image](https://github.com/user-attachments/assets/e920a2b1-7169-4562-b3ad-ad99ec5ae30e)

Reply:
Not to make this more complicated, but as a TODO item for the future, shouldn't _both_ modes be being tested?  This also means we should probably be testing / ensuring that if containerd is already running as a separate service that it is used by default (like it does on Linux), right?

Reply:
Agree with @tianon here.
This can be future work, though.